x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/header_actions.tsx

```diff
@@ -51,6 +51,24 @@ const SortingColumnsContainer = styled.div`
   }
 `;
 
+const FieldBrowserContainer = styled.div`
+  .euiToolTipAnchor {
+    .euiButtonContent {
+      padding: 0 4px;
```

Comment:
nit: consider using the equivalent  `paddingSizes.xs` (via `node_modules/@elastic/eui/dist/eui_theme_amsterdam_dark.json`)

```
padding: ${({ theme }) => `0 ${theme.eui.paddingSizes.xs}`};
```